soc/riscv/riscv-ite/it8xxx2/ilm.c

```diff
@@ -185,8 +185,13 @@ static const struct ilm_config ilm_config = {
 		SCAR_REG(14),
 		SCAR_REG(15),
 	}};
+#if defined(CONFIG_SOC_IT81202_CX) || defined(CONFIG_SOC_IT81302_CX)
+BUILD_ASSERT(ARRAY_SIZE(ilm_config.scar_regs) * ILM_BLOCK_SIZE == KB(60),
+	     "Maximum number of SCAR registers defined for 60k RAM size");
+#else
 BUILD_ASSERT(ARRAY_SIZE(ilm_config.scar_regs) * ILM_BLOCK_SIZE == KB(RAM_SIZE),
```

Comment:
It might be reasonable to remove this condition completely, if we expect all 81302 variants to have 60K ILM size.

Reply:
I agree, because the `scar_regs` array is fixed for all variants, you only need to check that the SCAR reg count maps to 60 KiB of RAM.

Reply:
This is a limitation of CX, other variants expect the size of ILM to be the same as the SRAM size.